src/bun.js/node/node_fs.zig

```diff
@@ -141,6 +143,185 @@ pub const Async = struct {
+
+                        bun.debugAssert(uv.uv_fs_read(loop, &task.req, fd, &.{B(args_.buffer.slice()[args_.offset..])}, 1, args_.position orelse -1, &uv_callback) == .zero);
+                        log("uv read({d}) = ~~", .{fd});
+                    },
+                    .write => {
+                        const args_: Arguments.Write = task.args;
+                        const B = uv.uv_buf_t.init;
+                        const fd = args_.fd.impl().uv();
+
+                        bun.debugAssert(uv.uv_fs_write(loop, &task.req, fd, &.{B(args_.buffer.slice()[args_.offset..])}, 1, args_.position orelse -1, &uv_callback) == .zero);
+                        log("uv write({d}) = ~~", .{fd});
+                    },
+                    .readv => {
+                        const args_: Arguments.Readv = task.args;
+                        const fd = args_.fd.impl().uv();
+                        const bufs = args_.buffers.buffers.items;
+                        const pos: i64 = args_.position orelse -1;
+
+                        var sum: u64 = 0;
+                        for (bufs) |b| sum += b.slice().len;
+
+                        bun.debugAssert(uv.uv_fs_read(loop, &task.req, fd, bufs.ptr, @intCast(bufs.len), pos, &uv_callback) == .zero);
+                        log("uv readv({d}, {*}, {d}, {d}, {d} total bytes) = ~~", .{ fd, bufs.ptr, bufs.len, pos, sum });
+                    },
+                    .writev => {
+                        const args_: Arguments.Writev = task.args;
+                        const fd = args_.fd.impl().uv();
+                        const bufs = args_.buffers.buffers.items;
+                        const pos: i64 = args_.position orelse -1;
+
+                        var sum: u64 = 0;
+                        for (bufs) |b| sum += b.slice().len;
+
+                        bun.debugAssert(uv.uv_fs_write(loop, &task.req, fd, bufs.ptr, @intCast(bufs.len), pos, &uv_callback) == .zero);
+                        log("uv writev({d}, {*}, {d}, {d}, {d} total bytes) = ~~", .{ fd, bufs.ptr, bufs.len, pos, sum });
+                    },
+                    else => comptime unreachable,
+                }
+
+                return task.promise.value();
+            }
+
+            fn uv_callback(req: *uv.fs_t) callconv(.C) void {
+                defer uv.uv_fs_req_cleanup(req);
+                const this: *Task = @ptrCast(@alignCast(req.data.?));
+                var node_fs = NodeFS{};
+                this.result = @field(NodeFS, "uv_" ++ @tagName(FunctionEnum))(&node_fs, this.args, @intFromEnum(req.result));
+
+                if (this.result == .err) {
+                    this.result.err.path = bun.default_allocator.dupe(u8, this.result.err.path) catch "";
+                    std.mem.doNotOptimizeAway(&node_fs);
+                }
+
+                this.globalObject.bunVMConcurrently().eventLoop().enqueueTaskConcurrent(JSC.ConcurrentTask.createFrom(this));
```

Comment:
Since `uv_callback` is run on the same thread, we don't need to use enqueueTaskConcurrent.

```suggestion
                this.globalObject.bunVM().eventLoop().enqueueTask(JSC.Task.init(this))
```